src/test/models/car.ts

```diff
@@ -1,6 +1,7 @@
 import * as mongoose from 'mongoose';
 
 import { Typegoose, prop, pre } from '../../typegoose';
+import {Decimal128} from 'bson';
```

Comment:
Just a cosmetic issue, but please add spaces after the opening and before the closing bracket.